src/test/java/io/confluent/avro/random/generator/IterationTest.java

```diff
@@ -62,4 +62,12 @@ public void shouldGenerateFromSameSchemaOnMultipleThreads() {
           generator.generate();
         });
   }
+
+  @Test
+  public void shouldSimulatePreviousIterations() {
+    for (long i = 0; i < 1e3; i++) {
```

Comment:
This notation is a little strange here considering it's used for floating-point literals. Don't think it adds much in readability, could we just use a vanilla numeric literal here (or maybe `1_000` if we want to be that fancy)?
```suggestion
    for (long i = 0; i < 1000; i++) {
```